src/main/steps/c100-rebuild/permission/template.njk

```diff
@@ -0,0 +1,19 @@
+{% extends "common/template.njk" %}
+
+{% block page_content %}
+
+    <h1 class="govuk-heading-xl">{{title}}</h1>
+    <p class="govuk-body">{{courtPermissionLine1}}</p>
+    <p class="govuk-body">{{courtPermissionLine2}}</p>
+    <ul class="govuk-list govuk-list--bullet govuk-!-margin-left-7 govuk-!-margin-bottom-6">
+        <li>{{courtPermissionBullet1}}</li>
+        <li>{{courtPermissionBullet2}}</li>
+        <li>{{courtPermissionBullet3}}</li>
+    </ul>
+    <p class="govuk-body">{{courtPermissionLine3}}</p>
```

Comment:
@ShakiraKP Can we use an Array for these labels in content.ts & a for lop in template.njk to render them ?

Reply:
Okay I will look into this, just for the bullet points or for all labels?

Reply:
completed